pkg/operator/controllers/genevalogging/genevalogging_controller.go

```diff
@@ -69,21 +63,13 @@ func (r *Reconciler) Reconcile(ctx context.Context, request ctrl.Request) (ctrl.
 	}
 
 	r.log.Debug("running")
-	mysec, err := r.kubernetescli.CoreV1().Secrets(operator.Namespace).Get(ctx, operator.SecretName, metav1.GetOptions{})
-	if err != nil {
-		return reconcile.Result{}, err
-	}
-
-	// TODO: dh should be a field in r, but the fact that it is initialised here
-	// each time currently saves us in the case that the controller runs before
-	// the SCC API is registered.
-	dh, err := dynamichelper.New(r.log, r.restConfig)
```

Comment:
I think this should not be a problem - if there is an error due to API not being registered or something - we just return an error and requeue with backoff.